### ISSUE TITLE
Deprecation Warning: image/bmp is not a valid content type. Rails 7.1

### ISSUE THREAD

**Issue by williantenfen** [igorkasyanchuk/active_storage_validations#164]
Couldn't get around this:
`DEPRECATION WARNING: image/bmp is not a valid content type, it should not be used when creating a blob, and support for it will be removed in Rails 7.1.`

Versions:
`Rails 7.0.4`
`active_storage_validations 0.9.8`

Spec:
`it { is_expected.to validate_content_type_of(:file).allowing("font/otf", "font/woff", "font/woff2")  }`

**Comment by crismali** [comment]
@williantenfen I was able to get around this warning by chaining a specific `rejecting` to the `allowing` like

```rb
it { is_expected.to(
  validate_content_type_of(:file)
    .allowing("font/otf", "font/woff", "font/woff2")
    .rejecting("application/pdf", "image/png") 
)}
```

The matcher falls back to `Marcel::TYPES.keys` when `rejecting` isn't chained so if you want things to behave like they do now but without the deprecated type you could try something like 

```rb
it { is_expected.to(
  validate_content_type_of(:file)
    .allowing("font/otf", "font/woff", "font/woff2")
    .rejecting(Marcel::TYPES.keys - ["font/otf", "font/woff", "font/woff2", "image/bmp"]) }
)}
```

**Comment by williantenfen** [comment]
@crismali Thanks, looks like it worked...

**Comment by HeshamaMohamed** [comment]
@igorkasyanchuk, I think this issue is fixed in https://github.com/igorkasyanchuk/active_storage_validations/pull/169

**Comment by Mth0158** [comment]
Looks resolved to me @igorkasyanchuk, I'll close it and we reopen it if necessary
